cmd/common-main.go

```diff
@@ -175,7 +175,9 @@ func minioConfigToConsoleFeatures() {
 	if globalMinioEndpoint != "" {
 		os.Setenv("CONSOLE_MINIO_SERVER", globalMinioEndpoint)
 	} else {
-		os.Setenv("CONSOLE_MINIO_SERVER", getAPIEndpoints()[0])
+		// Explicitly set 127.0.0.1 so Console will automatically bypass TLS verification to the local S3 API.
+		// This will save users from providing a certificate with IP or FQDN SAN that points to the local host.
+		os.Setenv("CONSOLE_MINIO_SERVER", fmt.Sprintf("%s://127.0.0.1:%s", getURLScheme(globalIsTLS), globalMinioPort))
 	}
```

Comment:
We can also remove documentation regarding MINIO_SERVER_URL @vadmeste ? // cc @ravindk89 FYI upcoming release.

Reply:
Yeah, I'm only sort of caught up to what the main changes are here where `MINIO_SERVER_URL` is no longer required.

If someone can summarize that would be helpful. We might need to keep the variable as a general reference for a little while noting it has been deprecated, removed, and no longer is necessary starting with that Release.

Reply:
MINIO_SERVER_URL is no longer honored or required @ravindk89 after this change.